tools/sql_extraction/src/main/kotlin/DataFlowEngine.kt

```diff
@@ -3,12 +3,16 @@ package com.google.cloud.sqlecosystem.sqlextraction
 import com.google.cloud.sqlecosystem.sqlextraction.output.Location
 import com.google.cloud.sqlecosystem.sqlextraction.output.QueryFragment
 import com.google.cloud.sqlecosystem.sqlextraction.output.QueryUsages
+import mu.KotlinLogging
+
+private val LOGGER = KotlinLogging.logger { }
```

Comment:
Is this a global logger? should this be inside the class?

Reply:
This was suggested by the [framework](https://github.com/MicroUtils/kotlin-logging/wiki#obtaining-a-logger) as the preferred way of obtaining a logger. Otherwise, it increases boilerplate code.